superglue/lib/reducers/index.ts

```diff
@@ -51,19 +73,19 @@ function saveResponse(state, pageKey, page) {
   const existingPage = state[pageKey]
 
   if (existingPage) {
-    page = addPlaceholdersToDeferredNodes(existingPage, page)
+    nextPage = addPlaceholdersToDeferredNodes(existingPage, nextPage)
   }
   constrainPagesSize(state)
-  state[pageKey] = page
+  state[pageKey] = nextPage
```

Comment:
Just using a local variable to type from VisitResponse to Page. Functionally the same

```diff
function saveResponse(state, pageKey, page) {
   state = Object.assign({}, state);
-  page = Object.assign({
+  var nextPage = Object.assign({
     pageKey: pageKey,
     fragments: []
   }, page, {
@@ -51,10 +50,10 @@ function saveResponse(state, pageKey, page) {
   });
   var existingPage = state[pageKey];
   if (existingPage) {
-    page = addPlaceholdersToDeferredNodes(existingPage, page);
+    nextPage = addPlaceholdersToDeferredNodes(existingPage, nextPage);
   }
   constrainPagesSize(state);
-  state[pageKey] = page;
+  state[pageKey] = nextPage;
   return state;
 }
```